README.rst

```diff
@@ -40,14 +40,57 @@ USAGE
 
 If you want to learn more about coala-bears, its functionality and its usage,
 please take a look at our
-`tutorial <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html#what-is-a-bear>`__.
+`tutorial <http://coala.readthedocs.org/en/latest/index.html>`__.
```

Comment:
It's part of the tutorial. Second part. Why should we assume that user already read the first one? I'll change that to http://coala.readthedocs.org/en/latest/Users/Tutorials/Tutorial.html